clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts

```diff
@@ -155,19 +155,19 @@ describe("Config wizard with plus settings", () => {
         const { body } = interception.request;
         expect(body.length).to.eql(numSystems);
       });
-      cy.getByTestId("systems-classify-table");
-      cy.url().should("contain", "classify-systems");
+      // cy.getByTestId("systems-classify-table");
+      // cy.url().should("contain", "classify-systems");
```

Comment:
These are tests for the "data flow scanner" and they verify functionality by navigating to the "Classify systems" page. However, @mfbrown asked that I remove the "Classify systems" page from the navbar as part of this ticket. For now I just removed the assertions that occur after navigating to the system-classify-table.